spring-cloud-stream-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -81,147 +71,586 @@ public class StreamApplicationTests {
 
-When scaling up Spring Cloud Stream applications, each instance can receive information about how many other instances
-of the same application exist and what its own instance index is. This is done through the
-`spring.cloud.stream.instanceCount` and `spring.cloud.stream.instanceIndex` properties. For example, if there are 3
-instances of the HDFS sink application, all three will have `spring.cloud.stream.instanceCount` set to 3, and the
-applications will have `spring.cloud.stream.instanceIndex` set to 0, 1 and 2, respectively. When Spring Cloud Stream
-applications are deployed via Spring Cloud Data Flow, these properties are configured automatically, but when Spring
-Cloud Stream applications are launched independently, these properties must be set correctly. By default
-`spring.cloud.stream.instanceCount` is 1, and `spring.cloud.stream.instanceIndex` is 0.
+While the concept of publish-subscribe messaging is not new, Spring Cloud Stream takes the extra step of making it an opinionated choice for its application model, and makes it easy for users to work with it across different platform by using the native support of the middleware. To go with it, two other concepts are central to Spring Cloud Stream: consumer groups and durability.
 
-Setting up the two properties correctly on scale up scenarios is important for addressing partitioning behavior in
-general (see below), and they are always required by certain types of binders (e.g. the Kafka binder) in order to
-ensure that data is split correctly across multiple consumer instances.
 
-==== Advanced Binding Properties
+[[consumer-groups]]
+==== Consumer Groups
+While the publish subscribe model ensures that it is easy to connect multiple application by sharing a topic, it is equally important to be able to scale up by creating multiple instances of a given application, which would find themselves in a competing consumer relationship with each other. Spring Cloud Stream models this behavior through the concept of a consumer group, which is similar to (and inspired by) the notion of consumer groups in Kafka. Each consumer binding can specify a group name such as `spring.cloud.stream.bindings.input.group=hdfsWrite` or `spring.cloud.stream.bindings.input.group=average` as shown in the picture. All groups that subscribe to a given destination will receive a copy of the published data, but, within a group, only one application will receive a given message. By default, which is the case when a group is not specified, Spring Cloud Stream assigns the application to an anonymous, independent, single-member consumer group. Otherwise said, if no consumer group is specified for a binding, it will be in a publish-subscribe relationship with all the other consumer groups.
```

Comment:
It's been meant as a follow-up to " By default, which is the case when a group is not specified, Spring Cloud Stream assigns the application to an anonymous, independent, single-member consumer group."